appeals/web/src/server/appeals/appeal-details/appellant-case/appellant-case.mapper.js

```diff
@@ -689,13 +689,15 @@ function generateCaseTypeSpecificComponents(
 export function getPageHeadingTextOverrideForFolder(folder) {
 	switch (folder.path.split('/')[1]) {
 		case APPEAL_DOCUMENT_TYPE.PLANS_DRAWINGS:
-			return 'supporting documents submitted with statement';
+			return 'Plans, drawings and list of plans';
```

Comment:
This could use constants that are prepended with the action (see above)

Reply:
Thanks, as mentioned in response to the related comment, I agree but I don't think this ticket is the correct place